bobtemplates/django/+package.fullname+/README.md.bob

```diff
@@ -0,0 +1,124 @@
+# {{{package.fullname}}}
+
+## Development
+
+### Setup
+
+A few steps are needed to setup the backend.
+Run these steps within the project folder.
+
+1. Create and activate a python virtual environment:
+    `python3 -m venv . && source bin/activate`
+
+2. Install the application and its requirements:
+    `pip install -e .[development]`
+
+3. Create a postgresql database
+    `createdb {{{package.fullname}}}_development`
+
+4. Run the migrations:
+    `./manage.py migrate`
+
+5. Run the server:
+    `./manage.py runserver`
+
+*optional steps*
+
+6. Create a super user:
+    `./manage.py createsuperuser`
+
+### Testing
+
+1. Activate the python virtual environment:
+    `source bin/activate`
+
+2. Install the requirements:
+    `pip install -e .[test]`
+
+3. Run the tests:
+    `tox`
+
```

Comment:
I'd probably also document how to run tests without tox (with `py.test`). This is a good alternative when developing.